notebooks/ch-applications/image-processing-frqi-neqr.ipynb

```diff
@@ -23626,7 +23626,7 @@
     }
    ],
    "source": [
-    "# Encode the third pixel whose value is (10101010):\n",
+    "# Encode the third pixel whose value is (11111111):\n",
```

Comment:
@bopardikarsoham - thank you for the spelling corrections.

For this particular change here, did you mean to include it? Or is it the same changes in [this pull request](https://github.com/Qiskit/platypus/pull/690/files)?

Reply:
This is actually from my previous PR #690 whose merge is pending